### PR TITLE
Update password variable term

### PR DESCRIPTION
**Pull request by melcorr**
fixes #7065

Make passwords consistent across Ansible installer and Pulp core docs.

### CODE REVIEW

**Comment by mdellweg** [comment]
I like the idea. However, i see another one inline 130
`   $ pulpcore-manager reset-admin-password --password admin`
where, for inconsistencies sake, we set the admin-password to `admin`.

**Comment by melcorr** [comment]
@fao89 @mikedep333 hey guys, @mdellweg makes an excellent point. With regards to the Ansible installer docs here and in the Ansible installer guide, would you be comfortable with us using the same <<>> around all variable values. Consistency is key.

File: docs/installation/instructions.rst [hunk 1]
```diff
@@ -42,7 +42,7 @@ Create your pulp_install.yml playbook to use with the installer (Uncomment the d
        pulp_settings:
          secret_key: secret
```
Comment (fao89) [inline docs/installation/instructions.rst:43]: we need the same here

**Comment by melcorr** [comment]
I am happy to implement feedback from @fao89 - but is there any chance that I could convince you guys of the original premise of the issue I opened: 

"Replace the Ansible installer section in the Pulpcore docs with just a description and redirect to the Pulp Ansible Installer docs"

Unless we are stricter with the docs or figure out a way to have a single source across different doc sites, there will be a lot of inconsistencies. If we link to the primary and most-likely-to-be-up-to-date docs, I think that we might avoid future confusion.

WDYT?

**Comment by mdellweg** [comment]
Absolutely, if we have documentation for the ansible installer somewhere, we should not even try to keep a (usually outdated) copy of it on the main website. I think, this is the purpose, hyperlinks were invented for.

**Comment by fao89** [comment]
> I am happy to implement feedback from @fao89 - but is there any chance that I could convince you guys of the original premise of the issue I opened:
> 
> "Replace the Ansible installer section in the Pulpcore docs with just a description and redirect to the Pulp Ansible Installer docs"
> 
> Unless we are stricter with the docs or figure out a way to have a single source across different doc sites, there will be a lot of inconsistencies. If we link to the primary and most-likely-to-be-up-to-date docs, I think that we might avoid future confusion.
> 
> WDYT?

I agree with you, just for sharing the context, this brief explanation came from here:
https://pulp.plan.io/issues/6658
By that time we didn't have https://pulp-installer.readthedocs.io/en/latest/

**Comment by mikedep333** [comment]
> I am happy to implement feedback from @fao89 - but is there any chance that I could convince you guys of the original premise of the issue I opened:
> 
> "Replace the Ansible installer section in the Pulpcore docs with just a description and redirect to the Pulp Ansible Installer docs"
> 
> Unless we are stricter with the docs or figure out a way to have a single source across different doc sites, there will be a lot of inconsistencies. If we link to the primary and most-likely-to-be-up-to-date docs, I think that we might avoid future confusion.
> 
> WDYT?

I can go either way. I think a version of the Installer quickstart in our main docs (like we do now) has benefit, but it also increases our maintenance, and makes people more likely to overlook our Installer docs.

File: CHANGES/7065.doc [hunk 2]
```diff
@@ -0,0 +1 @@
+Make password variable consistent with Ansible installer example playbook
```
Comment (lubosmj) [inline CHANGES/7065.doc:1]:
```suggestion
Made password variable consistent with Ansible installer example playbook
```

We decided to use the past simple tense for changelog messages. Refer to https://www.mail-archive.com/pulp-dev@redhat.com/msg03803.html to learn more.

**Comment by melcorr** [comment]
I am so sorry that it took so long for me to get back to this. 
It kept getting pushed down my list. 
As we discussed in the previous sprint planning, I am OK with leaving this as-is for now and if we get several complaints about inconsistencies etc, we can then maybe revisit having only one source of truth.

**Pull request merged**